internal/handlers/handlers.go

```diff
@@ -1 +1,74 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func HandleMain(res http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodGet {
+		http.Error(res, fmt.Sprint("The server does not support the method", req.Method), http.StatusInternalServerError)
```

Comment:
Есть специальная для этого ошибка
```suggestion
		http.Error(res, fmt.Sprint("The server does not support the method", req.Method), http.StatusMethodNotAllowed)
```